sql/_30_banana_pie_qa/issue_20158_string_compression/cases/trigger_view.sql

```diff
@@ -0,0 +1,70 @@
+--+ holdcas on;
+
+-- trigger
+drop table if exists t1;
+
+drop table if exists t2;
+
+create table t1(i1 int,s1 varchar);
+
+create table t2(s1 varchar);
+
+create trigger tri_ti before statement insert on t2 execute insert into t1 values(1,repeat('bcd',200));
```

Comment:
Please drop the trigger tri_ti and the vie v_t3. 
And also check others in this test case.